packages/ocs/dashboards/common/capacity-card/capacity-card.scss

```
@@ -1,4 +1,4 @@
-@import '@odf/shared/styles/skeleton';
+@use '@odf/shared/styles/skeleton' as *;
```

The main intention of  `@use` is to use namespaces (rather than global namespace) to avoid potential collisions:
```suggestion
@use '@odf/shared/styles/skeleton' as sk;
```